components/layout/text.rs

```diff
@@ -165,8 +166,8 @@ impl TextRunScanner {
                     white_space::T::pre_line => CompressionMode::CompressWhitespace,
                 };
                 text_transform = inherited_text_style.text_transform;
-                letter_spacing = inherited_text_style.letter_spacing.0;
-                word_spacing = inherited_text_style.word_spacing.0
+                letter_spacing = inherited_text_style.letter_spacing;
+                word_spacing = inherited_text_style.word_spacing.value().cloned()
```

Comment:
Yes, won't the whole thing be`Copy`? `&T` is copy, as is `Option<&T>`. Why is a `cloned()` necessary?

(Generally best to avoid explicit cloning copy things, signals more cost than there actually is)